src/modules/fancyzones/FancyZonesLib/FancyZonesLib.vcxproj

```diff
@@ -147,7 +147,9 @@
     </ProjectReference>
     <ResourceCompile Include="Generated Files/fancyzones.rc" />
     <None Include="packages.config" />
-    <None Include="Resources.resx" />
+    <None Include="Resources.resx">
+      <SubType>Designer</SubType>
```

Comment:
This looks like an unintended addition. Can we please revert it?

Reply:
Yep, Visual Studio did that. Will do.